client/dashboard/domains/domain-contact-details/contact-form.tsx

```diff
@@ -29,16 +29,10 @@ interface ContactFormProps {
 	domainName: string;
 	initialData?: DomainContactDetails;
 	onSubmit?: ( data: DomainContactDetails ) => void;
-	onCancel?: () => void;
 	errors?: Partial< Record< keyof DomainContactDetails, string > >;
```

Comment:
Same as the above comment. Do we need the `errors` prop?

Reply:
Good catch, removed as well :)